util/version/Cargo.toml

```diff
@@ -8,9 +8,9 @@ authors = ["Parity Technologies <admin@parity.io>"]
 build = "build.rs"
 
 [package.metadata]
-# This versions track. Should be changed to `stable` or `beta` when on respective branches.
+# This versions track. Should be changed to `stable` when on respective branches.
 # Used by auto-updater and for Parity version string.
-track = "nightly"
+track = "stable"
```

Comment:
Why are we merging this to master? Shouldn't master be on "nightly" track and v2.7.0-stable on "stable"?

Reply:
Yes correct - this change should be reverted, and only applied when we port this PR to the `v2.7.0-stable` branch.